src/runtime_src/doc/toc/fpga_device_ready.rst

```diff
@@ -0,0 +1,54 @@
+.. _fpga_device_ready.rst:
+
+
+FPGA device readiness within user VM
+************************************
+
+To make the FPGA in user VM ready to use, the mgmt side has to be ready first. Beside that, depending on how
+the FPGA is deployed, there are still other factors to be considered.
+
+With MSD/MPD or Plugins used
+============================
+
+In this case, :ref:`mailbox.main.rst` has to be opened from user, and  MPD service must be active.
+
+For MSD/MPD, the MSD service must be active also either from hypervisor or mgmt service domain (VM)
+
+For Plugins, the vendor specific plugin package has to be installed
+
+Without MSD/MPD or Plugins used
+===============================
+  
+In this case, :ref:`mailbox.main.rst` is still being used, but there is no open to the mailbox subdevice from user,
+and MPD service is inactive.
+To make the FPGA device ready, the XCL_MAILBOX_REQ_LOAD_XCLBIN opcode has to be disabled from xclmgmt like following, 
```

Comment:
disabled from xclmgmt --> disabled in xclmgm

Reply:
changed